go/interpreter/lfvm/interpreter.go

```diff
@@ -636,17 +636,48 @@ func steps(c *context, oneStepOnly bool) {
 
 func satisfiesStackRequirements(c *context, op OpCode) bool {
 	stackLen := c.stack.len()
-	if stackLen < staticStackBoundary[op].stackMin {
+
+	// It would be necessary here to check that the operation index is not out
+	// of bounds for the precomputed stack limits. However, this adds 15%
+	// overhead to this function. To avoid this, the _precomputedStackLimits
+	// is padded with a stackLimits{0, 0} at the end up to 256 elements. This
+	// way, by only taking the last 8 bits of the OpCode, the index is always
+	// within bounds.
```

Comment:
I guess there will be a loss here if I manage to finish the opcodes refactor, lfvm extended instructions will be > 0xff. 
Any future plans here?

Reply:
Yes, use 512 elements and the mask "0x1ff".